homework9/src/components/Input/InputApply.jsx

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+
+import Input from "./Input";
+
+import "./input.style.css";
+
+const InputApply = ({
+  title,
+  value,
+  type,
+  options,
+  onChange,
+  placeholder,
+  isDisabled,
```

Comment:
for that case u can get only `title` and `otherProps`
{
title,
...otherProps,
}
    <Input
      {...otherProps}
    />
and then